kie-dmn-core/src/main/java/org/kie/dmn/core/compiler/DMNEvaluatorCompiler.java

```diff
@@ -322,6 +327,63 @@ private DMNExpressionEvaluator compileDecisionTable(DMNCompilerContext ctx, DMNM
         DMNDTExpressionEvaluator dtee = new DMNDTExpressionEvaluator( node, dtf );
```

Comment:
@tarilabs we have to check if the hit policy is PRIORITY and no output values are defined, then we have to raise a compilation error. With other hit policies, the output values are optional, but with PRIORITY, they are mandatory.